src/Cake.Common/Tools/DotNetCore/DotNetCoreAliases.cs

```diff
@@ -0,0 +1,522 @@
+        /// <param name="project">The projects path.</param>
+        /// <example>
+        /// <code>
+        ///     DotNetCorePublish("./src/*");
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Publish")]
+        [CakeNamespaceImport("Cake.Common.Tools.DotNetCore.Publish")]
+        public static void DotNetCorePublish(this ICakeContext context, string project)
+        {
+            context.DotNetCorePublish(project, null);
+        }
+
+        /// <summary>
+        /// Publish all projects.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="project">The projects path.</param>
+        /// <param name="settings">The settings.</param>
+        /// <example>
+        /// <code>
+        ///     var settings = new DotNetCorePublishSettings
+        ///     {
+        ///         Framework = "dnxcore50",
+        ///         Configuration = "Release",
+        ///         OutputDirectory = "./artifacts/"
+        ///     };
+        ///
+        ///     DotNetCorePublish("./src/*", settings);
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Publish")]
+        [CakeNamespaceImport("Cake.Common.Tools.DotNetCore.Publish")]
+        public static void DotNetCorePublish(this ICakeContext context, string project, DotNetCorePublishSettings settings)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException("context");
+            }
+
+            if (settings == null)
+            {
+                settings = new DotNetCorePublishSettings();
+            }
+
+            var restorer = new DotNetCorePublisher(context.FileSystem, context.Environment, context.ProcessRunner, context.Globber);
+            restorer.Publish(project, settings);
+        }
+
+        /// <summary>
+        /// Test project.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <example>
+        /// <code>
+        ///     DotNetCoreTest();
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Test")]
+        [CakeNamespaceImport("Cake.Common.Tools.DotNetCore.Test")]
+        public static void DotNetCoreTest(this ICakeContext context)
+        {
+            context.DotNetCoreRun(null, null);
+        }
+
+        /// <summary>
+        /// Test project with path.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="project">The project path.</param>
+        /// <example>
+        /// <code>
+        ///     DotNetCoreTest("./src/Project");
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Test")]
+        [CakeNamespaceImport("Cake.Common.Tools.DotNetCore.Test")]
+        public static void DotNetCoreTest(this ICakeContext context, string project)
```

Comment:
Is this for testing a specific project, or does this method accept a glob pattern such as `./src/*/Project`?